sigridci/gitconfig-add-safe-directory

```diff
@@ -0,0 +1,2 @@
+[safe]
+	directory = *
```

Comment:
This line now seems to start with a tab character instead of 4 spaces, was that intentional? I'm not entirely sure if that just works? Out of curiosity, why not just running the `git config --global --add safe.directory '*'` command so it will generate a config file that it will like for sure?

Reply:
Ah yes, that's better! The reason I arrived at this solution is [this post](https://stackoverflow.com/a/71904131) that suggested some issues with running the command on different platforms, but that was when I still had the idea to run the command in the Python script 😬